### PR TITLE
CANVAS-120 - Analytics change columns

### PR DESCRIPTION
**Pull request by finnzink**

### CODE REVIEW

File: src/server/api/index.js [hunk 1]
```diff
@@ -2,9 +2,7 @@ require('dotenv').config();
 const express = require('express');
 const oauth = require('oauth-sign');
 const btoa = require('btoa');
-const json2xls = require('json2xls');
```
Comment (rlorenzo) [inline src/server/api/index.js:5]: @finnzink If you are no longer using json2xls, make sure to delete it from the package list as well.

yarn remove json2xls

File: src/server/api/index.js [hunk 2]
```diff
@@ -2,9 +2,7 @@ require('dotenv').config();
 const express = require('express');
 const oauth = require('oauth-sign');
 const btoa = require('btoa');
-const json2xls = require('json2xls');
-const fs = require('fs');
```
Comment (rlorenzo) [inline src/server/api/index.js:6]: @finnzink Same with these other packages if they are no longer needed

Comment (finnzink) [reply src/server/api/index.js:6]: done. fs is used elsewhere so it needs to be kept in

**Review by kilometerskang — APPROVED** [review]
Looks good to me, also tested and it works

**Pull request merged**